documentation/kyverno-cli.md

```diff
@@ -5,9 +5,16 @@
 
 The Kyverno Command Line Interface (CLI) is designed to validate policies and test the behavior of applying policies to resources before adding the policy to a cluster. It can be used as a kubectl plugin and as a standalone CLI.
 
-## Installation
+## Build the CLI
 
-You can download the CLI binary for your OS from the releases page [here](https://github.com/nirmata/kyverno/releases).
+You can build the CLI binary for your OS by running the command:
+
+```bash
+git clone https://github.com/nirmata/kyverno.git
+cd github.com/nirmata/kyverno
+make cli
+cd ./cmd/cli/kubectl-kyverno/ 
```

Comment:
Does make CLI run `go get`? If so, it will place the CLI binary in the $GOPATH/bin directory. We can mention that, or ask them to move the CLI to a  directory in their path. See example at: https://downloads.nirmata.io/nctl/downloads/#installation

Reply:
No, it only builds the CLI binary. 

Adding the instruction to move it to the user's PATH.